ktbridge/README.md

```diff
@@ -19,7 +19,8 @@ kotlin.js.compiler=ir
 Define a Shared API
 -------------------
 
-Create an interface and value types that'll be shared between platforms.
+In your `commonMain` directory, create an interface and value types that'll be shared between
+platforms. Use an `expect` val to declare a shared global instance.
```

Comment:
A global instance will not work for us. We have multiple VMs and can reload a VM based on feature flags or new copies of the JS / bytecode being downloaded without restarting the app.

Reply:
Even on the JS-side I don't think it's safe to assume a single, global instance in practice.